app/controllers/passengers_controller.rb

```diff
@@ -27,6 +27,13 @@ def index
       @passengers = @passengers.active
       @filters << 'active'
     end
+
+    if( params[:print].present? )
+      pdf = PassengersPDF.new(@passengers, @filters)
+      send_data pdf.render, filename: "Going to be the date",
```

Comment:
Is this the final filename you wanted for this PDF?

Reply:
Yes..... Is that bad?

Reply:
You want the filename to be "Going to be the date" ?